Server/src/controllers/getBasicDataController.ts

```diff
@@ -0,0 +1,47 @@
+import { Request, Response } from 'express';
+import { retrieveBasicData } from '../services/getBasicDataService';
+
+export class getBasicDataController {
+    constructor() {
+    }
+    /**
+     * This controller will take a request, verify that it is valid, and if it valid will then process the request,
+     * send it to the getDataService to acquire an array of data sets that match the entered search terms, and
+     * then return a response with the array of data sets. If the request is invalid, or some other problem arrises, 

```

Comment:
Usually comments about the class goes above the class name

Reply:
Comments fixed